packages/wp-story-editor/src/style.css

```diff
@@ -26,6 +26,11 @@
   display: none;
 }
 
+body.js.edit-story #wpcontent {
+  position: fixed;
+  width: calc(100% - 36px);
```

Comment:
This `width` won't be correct when the admin menu nav is expanded since `MENU_WIDTH` is 160 px whereas `MENU_FOLDED_WIDTH` is 36 px

Reply:
Okay gotcha Ill look into that